addons/html_builder/static/src/sidebar/option_container.xml

```diff
@@ -25,17 +25,25 @@
                             t-att-title="button.title" t-att-aria-label="button.title"
                             t-on-click="() => button.handler(props.editingElement)"/>
                 </t>
-                <t t-if="props.isClonable">
-                    <button class="fa fa-fw fa-clone oe_snippet_clone o_we_link o_we_hover_success btn btn-outline-success border-0 px-0 me-1"
-                            title="Duplicate this block"
-                            aria-label="Duplicate this block"
-                            t-on-click="cloneElement"/>
+                <t t-if="props.isClonable || props.cloneDisabledReason">
```

Comment:
We maybe want to merge isClonable and cloneDisabledReason ?

Reply:
We want the button to be present in this case, while `not isClonable` is about removing the button, so I kept them distinct.